src/main/kotlin/car/racing/GameRecords.kt

```diff
@@ -1,13 +1,29 @@
 package car.racing
 
 class GameRecords {
-    private val _records = mutableListOf<List<Int>>()
-    val records: List<List<Int>>
+    private val _records = mutableListOf<List<Record>>()
+    val records: List<List<Record>>
         get() = _records
```

Comment:
불변한 List 타입으로만 public하게 노출시키더라도 records를 다시 MutableList로 형변환하면 원본을 수정할 수 있습니다.
```suggestion
    private val _records = mutableListOf<List<Record>>()
    val records: List<List<Record>>
        get() = _records.toList()
```
이렇게 방어적 복사를 활용하시면 원본 데이터를 수정할 수 없습니다!